### PR TITLE
Gro model example

### PR DESCRIPTION
**Pull request by tolothy**
Showing a minimal working/not working example to demo our problem

### CODE REVIEW

File: MLproject [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: yiqing-test
+
+docker_env:
+  image: grointelligence/worker
```
Comment (MichaelWakahe) [inline MLproject:5]: For someone to pull this docker image, they would need to log into DockerHub with Gro credentials.

File: cluster-spec.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "spark_version": "7.3.x-scala2.12",
+  "num_workers": 1,
+  "node_type_id": "i3.xlarge"
```
Comment (MichaelWakahe) [inline cluster-spec.json:4]: Is there anywhere to specify the lifecycle of the instance as "spot"? This would save $

File: conda.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: yiqing-test
+channels:
+  - defaults
+dependencies:
+  - numpy>=1.14.3
```
Comment (MichaelWakahe) [inline conda.yaml:5]: I could recommend mentioning the minimum version of Python required in the README .

File: MLproject (hunk 1 above)
Comment (tolothy) [reply MLproject:5]: Let me add a comment here to make that clear, so we know this is a hard constraint.

File: cluster-spec.json (hunk 2 above)
Comment (tolothy) [reply cluster-spec.json:4]: Looks like doable, see https://docs.databricks.com/clusters/cluster-config-best-practices.html#on-demand-and-spot-instances. We can ask them when showing this example.

File: conda.yaml (hunk 3 above)
Comment (tolothy) [reply conda.yaml:5]: The conda configuration here is to show that the mlflow run without an issue when we configure the project using conda_env. For our use case with docker_env, this should have been specified by the docker container.

**Review by MichaelWakahe — APPROVED** [review]

**Pull request merged**